README.md

```diff
@@ -16,3 +16,14 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+“I certify that I have listed all sources used to complete this exercise, including the use
+of any Large Language Models. All of the work is my own, except where stated
+otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is
+suspected, charges may be filed against me without prior notice.”-Andrew Thomas
+
+# Answer 
+
+The best way to verify if the claim for time complexity true or not is to directly compare the time it takes for the algorithm to sort an arbitrary list against it's theoretical bounds. We only know that this ideal that this graph would be linear, as in some sort of straight line. We can't say what kind of line but if it's linear at all then we know it meets the criteria. To test this we would create a timer function and run the black-box algorithm against all permuations for some given list ensuring the best case worst case and average cases can be tested. We can then look at the output and see how closely it follows to our a linear trend. If it is some sort of straight line we know that it verifies the claim, if not then it couldn't be true. More specifically if the claim is true, then it's output should be linear. That is to say, if our output doesn't give us back data that's a linear line then it couldn't be the case that it has a general time compelxity of $O(n)$.
```

Comment:
You cannot just compare best and worst case for an input -- this tells you nothing about the complexity.